environments/mcdTransformers.toml

```diff
@@ -3,167 +3,9 @@
-    "jug_drip",
-    "jug_file_base",
-    "jug_file_ilk",
-    "jug_file_vow",
-    "jug_init",
-    "log_median_price",
-    "log_value",
-    "median_diss_batch",
-    "median_diss_single",
-    "median_drop",
-    "median_kiss_batch",
-    "median_kiss_single",
-    "median_lift",
-    "new_cdp",
-    "osm_change",
-    "pot_cage",
-    "pot_drip",
-    "pot_exit",
-    "pot_file_dsr",
-    "pot_file_vow",
-    "pot_join",
-    "rely",
-    "spot_file_mat",
-    "spot_file_par",
-    "spot_file_pip",
-    "spot_poke",
-    "tend",
-    "tick",
-    "vat_deny",
-    "vat_file_debt_ceiling",
-    "vat_file_ilk",
-    "vat_flux",
-    "vat_fold",
-    "vat_fork",
-    "vat_frob",
-    "vat_grab",
-    "vat_heal",
-    "vat_hope",
-    "vat_init",
-    "vat_move",
-    "vat_nope",
-    "vat_rely",
-    "vat_slip",
-    "vat_suck",
-    "vow_fess",
-    "vow_file_auction_attributes",
-    "vow_file_auction_address",
-    "vow_flog",
-    "vow_heal",
-    "yank",
-  ]
+  transformerNames = ["cat_v1_0_0", "cat_v1_1_0", "cdp_manager", "flap_v1_0_0", "flap_v1_0_9", "flip_bat_a_v1_0_0", "flip_bat_a_v1_0_9", "flip_bat_a_v1_1_0", "flip_comp_a_v1_1_2", "flip_eth_a_v1_0_0", "flip_eth_a_v1_0_9", "flip_eth_a_v1_1_0", "flip_eth_b_v1_1_3", "flip_knc_a_v1_0_8", "flip_knc_a_v1_0_9", "flip_knc_a_v1_1_0", "flip_link_a_v1_1_2", "flip_lrc_a_v1_1_2", "flip_mana_a_v1_0_9", "flip_mana_a_v1_1_0", "flip_paxusd_a_v1_1_1", "flip_sai_v1_0_0", "flip_tusd_a_v1_0_7", "flip_tusd_a_v1_0_9", "flip_tusd_a_v1_1_0", "flip_usdc_a_v1_0_4", "flip_usdc_a_v1_0_9", "flip_usdc_a_v1_1_0", "flip_usdc_b_v1_0_7", "flip_usdc_b_v1_0_9", "flip_usdc_b_v1_1_0", "flip_usdt_a_v1_1_1", "flip_wbtc_a_v1_0_6", "flip_wbtc_a_v1_0_9", "flip_wbtc_a_v1_1_0", "flip_zrx_a_v1_0_8", "flip_zrx_a_v1_0_9", "flip_zrx_a_v1_1_0", "flop_v1_0_1", "flop_v1_0_9", "jug", "median_bat", "median_comp", "median_eth", "median_knc", "median_link", "median_lrc", "median_mana", "median_usdt", "median_wbtc", "median_zrx", "pot", "spot", "vat", "vow", "auction_file", "bite", "cat_claw", "cat_file_box", "cat_file_chop_lump_dunk", "cat_file_flip", "cat_file_vow", "deal", "dent", "deny", "flap_kick", "flip_file_cat", "flip_kick", "flop_kick", "jug_drip", "jug_file_base", "jug_file_ilk", "jug_file_vow", "jug_init", "log_median_price", "log_value", "median_diss_batch", "median_diss_single", "median_drop", "median_kiss_batch", "median_kiss_single", "median_lift", "new_cdp", "osm_change", "pot_cage", "pot_drip", "pot_exit", "pot_file_dsr", "pot_file_vow", "pot_join", "rely", "spot_file_mat", "spot_file_par", "spot_file_pip", "spot_poke", "tend", "tick", "vat_deny", "vat_file_debt_ceiling", "vat_file_ilk", "vat_flux", "vat_fold", "vat_fork", "vat_frob", "vat_grab", "vat_heal", "vat_hope", "vat_init", "vat_move", "vat_nope", "vat_rely", "vat_slip", "vat_suck", "vow_fess", "vow_file_auction_attributes", "vow_file_auction_address", "vow_flog", "vow_heal", "yank", "flip_bal_a_v1_1_14", "median_bal", "flip_yfi_a_v1_1_14", "median_yfi"]
```

Comment:
I'm pretty torn about this diff. On the one hand, I like having these vals sorted/on new lines. On the other hand, that seems like a massive waste of time if we have to do it every time we run the generator. Wonder if it's worth creating a story for creating a script to format our toml (at a glance, doesn't seem like there's a widely used tool for doing this)

Reply:
I mentioned earlier - but I'd like to see them put in alphabetical order. Then you'd only have a diff on the new entries. I think that'd probably solve the issue.

In either case this should get fixed at the generator.

Reply:
Makes sense - do you know if we have a story for that? Are you thinking leave as is for now and then after we update and run the generator that it should take care of this?

Reply:
I was thinking I'd leave it as-is for now and we'd fix it in the generator. Not sure if we have a story, I'll have to check. I don't really feel strongly about how we fix it, whatever is easiest to just make clean to read diffs.